src/memory.rs

```diff
@@ -88,20 +79,28 @@ impl Memory {
         old_value
     }
 
-    pub fn set_bytes(&mut self, Address(start): Address, values: &[u8]) {
-        let start = start as usize;
+    pub fn set_bytes(&mut self, start: Address, values: &[u8]) {
+        let start = start.to_usize();
 
         // This panics if the range is invalid
```

Comment:
Nice one! Would this still panic if the range is invalid? Don't think so. In this case we could remove the warning.

Reply:
I've added a test to ensure that it still panics.

Reply:
Thanks! I think panicking is the only sane thing we can do here. 👍